snakemd/generator.py

```diff
@@ -660,7 +687,7 @@ class MDList(Element):
         set to True to render an ordered list (i.e., True -> 1. item)
     """
 
-    def __init__(self, items: Iterable[Union[str, InlineText, Paragraph, MDList]], ordered: bool = False) -> None:
+    def __init__(self, items: Iterable[Union[str, InlineText, Paragraph, MDList, CheckBox]], ordered: bool = False) -> None:
```

Comment:
```suggestion
    def __init__(self, items: Iterable[Union[str, InlineText, Paragraph, MDList]], ordered: bool = False) -> None:
```

I'm not sure this is needed since InlineText is already there.